jp/components/exporter_excel.md

```diff
@@ -148,266 +148,23 @@ this.excelExportService.onColumnExport.subscribe((args: IColumnExportingEventArg
 this.excelExportService.export(this.igxGrid1, new IgxExcelExporterOptions("ExportedDataFile"));
 ```
 
-`IgxGrid` からのデータのエクスポートで、エクスポート処理は行フィルタリングおよび列の非表示などの機能に応じてグリッドで表示されるデータのみをエクスポートします。`IgxExcelExporterOptions` オブジェクトのプロパティを設定し、エクスポーター サービスを構成してフィルターした行または非表示の列を含むことができます。このプロパティは以下の表で説明します。
+[**IgxGrid**](grid.md) からのデータのエクスポートで、エクスポート処理は行フィルタリングおよび列の非表示などの機能に応じてグリッドで表示されるデータのみをエクスポートします。[`IgxExcelExporterOptions`]({environment:angularApiUrl}/classes/igxexcelexporteroptions.html) オブジェクトのプロパティを設定し、エクスポーター サービスを構成してフィルターした行または非表示の列を含むことができます。このプロパティは以下の表で説明します。
 
 #### API まとめ
 
 以下は、その他の Excel Exporter サービスの API です。
 
-##### 入力
+The Excel Exporter service has a few more APIs to explore, which are listed below.
 
-以下の入力は **IgxExcelExporterService** コンポーネントで利用できます。
+* [`IgxExcelExporterService API`]({environment:angularApiUrl}/classes/igxexcelexporterservice.html)
+* [`IgxExcelExporterOptions API`]({environment:angularApiUrl}/classes/igxexcelexporteroptions.html)
 
-| 名前 | 型 | 説明 |
-| :--- | :--- | :--- |
-| `export` | Method | IgxGrid コンポーネントのデータを MS Excel 形式にエクスポートするメソッド。 |
-| `exportData` | Method | 配列データを MS Excel 形式でエクスポートするメソッド。 |
+使用したその他のコンポーネント:
 
-<div class="divider--half"></div>
-
-以下の入力は **IgxExcelExporterOptions** コンポーネントで利用できます。
-
-| 名前 | 型 | 説明 |
-| :--- | :--- | :--- |
-| `fileName` | string | エクスポート操作でファイル名を提供します。 |
-| `ignorePinning` | boolean | 列固定が無視されるかどうかを指定します。ignoreColumnsOrder を true に設定した場合、このオプションは常に true に設定されているとみなされます。 |
-| `ignoreColumnsOrder` | boolean | エクスポーターがグリッドの列順序を無視するかどうかを指定します。 |
-| `ignoreColumnsVisibility` | boolean | 非表示列をエクスポートするかどうかを指定します。 |
-| `ignoreFiltering` | boolean | フィルター アウトされた行をエクスポートするかどうかを指定します。 |
-| `ignoreSorting` | boolean | エクスポートするデータをグリッドのように並べ替えるかどうかを指定します。 |
-| `exportAsTable` | boolean | エクスポートするデータを Excel テーブルとして書式設定するかどうかを指定します。(デフォルトで True) |
-| `columnWidth` | number | エクスポートする Excel ファイルの列幅を設定します。指定なしまたは 0 の場合、列の最大文字列の幅が使用されます。 |
-| `rowHeight` | number | エクスポートされた Excel ファイルの行の高さを設定します。指定なしまたは 0 の場合、Excel 行のデフォルトの高さが使用されます。 |
-
-<div class="divider"></div>
-
-##### 出力
-
-以下の出力は **IgxExcelExporterService** コンポーネントで利用できます。
-
-| 名前 | 型 | 説明 |
-| :--- | :--- | :--- |
-| `onColumnExport` | EventEmitter<`IColumnExportingEventArgs`> | Excel 列がエクスポートされているときに発生します。 |
-| `onRowExport` | EventEmitter<`IRowExportingEventArgs`> | Excel 行がエクスポートされているときに発生します。 |
-| `onExportEnded` | EventEmitter<`IExcelExportEndedEventArgs`> | Excel ファイルがエクスポートされたときに発生します。 |
+* [**IgxGrid**](grid.md)
```

Comment:
Provide Typedoc and Sassdoc references to the IgxGridComponent API and IgxGrid theme (#function-igx-grid-theme), respectfully.

Look at the request change for the English version

Reply:
Fixed